docker-neo4j/azuredeploy.json

```diff
@@ -0,0 +1,399 @@
+                "type": "string",
+                "metadata": {
+                    "description": "Unique DNS Name for the Public IP used to access the Virtual Machine."
+                }
+            },
+            "vmName": {
+                "type": "string",
+                "defaultValue": "neodockervm",
+                "metadata": {
+                    "description": "Name of the sample VM to create"
+                }
+            },
+            "vmSize": {
+                "type": "string",
+                "defaultValue": "Standard_A1",
+                "allowedValues": [
+                    "Standard_A1",
+                    "Standard_A2",
+                    "Standard_A3",
+                    "Standard_A6",
+                    "Standard_A7",
+                    "Standard_A8",
+                    "Standard_A9",
+                    "Standard_A10",
+                    "Standard_A11",
+                    "Standard_D2",
+                    "Standard_D3",
+                    "Standard_D4",
+                    "Standard_D11",
+                    "Standard_D12",
+                    "Standard_D13",
+                    "Standard_D14"
+                ],
+                "metadata": {
+                    "description": "Size of the Virtual Machine."
+                }
+            },
+            "adminUsername": {
+                "type": "string",
+                "metadata": {
+                    "description": "Username for sample VM"
+                }
+            },
+            "adminPassword": {
+                "type": "securestring",
+                "metadata": {
+                    "description": "User password for sample VM"
+                }
+            },
+            "newStorageAccountName": {
+                "type": "string",
+                "metadata": {
+                    "description": "Stoarge Account Name for VM Disk"
+                }
+            },
+            "storageAccountType": {
+                "type": "string",
+                "allowedValues": [
+                    "Standard_LRS",
+                    "Standard_GRS",
+                    "Standard_RAGRS",
+                    "Premium_LRS"
+                ],
+                "metadata": {
+                    "description": "The type of the Storage Account created"
+                },
+                "defaultValue": "Standard_LRS"
+            },
+            "networkSecurityGroupName": {
+                "type": "string",
+                "metadata": {
+                    "description": "Name of the network security group."
+                },
+                "defaultValue": "myNSG"
+            },
+            "networkInterfaceName": {
+                "type": "string",
+                "metadata": {
+                    "description": "Name of the network security group."
+                },
+                "defaultValue": "myVMNic"
+            },
+            "publicIPAddressName": {
```

Comment:
This can become a variable to reduce the number of parameters a user has to enter